clang/test/CodeGen/cheri/riscv/cheriot-cap-import-attr.c

```diff
@@ -0,0 +1,22 @@
+// RUN: %clang_cc1 %s -o - "-triple" "riscv32cheriot-unknown-unknown" "-emit-llvm" "-mframe-pointer=none" "-mcmodel=small" "-target-abi" "cheriot" "-Oz" "-Werror" -std=c2x | FileCheck %s
+
+struct Uart {};
+
+// CHECK: @uart = external addrspace(200) global %struct.Uart, align 1 #0 
+__attribute__((cheriot_mmio("uart", "RWcm"))) extern struct Uart uart;
```

Comment:
Please add Sema tests for ill-formed versions of these.

Reply:
4ffcc2cb376e395259065c8636b29833bfce7139  adds the Sema [tests](https://github.com/CHERIoT-Platform/llvm-project/pull/147/commits/4ffcc2cb376e395259065c8636b29833bfce7139#diff-856b6784adca6f1c09cd4e735e28ccb06ec6832cd5eb01ab371b4aedb8b6cf1aR1-R12) for ill-formed versions of the attributes. Maybe there's a way to have tests on emitted IR and errors/warnings in the same file, but I was not able to make it work.

Reply:
I think it's fine and probably even good practice to separate them anyways.